charts/kubewarden-controller/Chart.yaml

```
@@ -20,7 +20,7 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.2.8
+version: 1.3.0
```

```suggestion
version: 1.2.8
```
We are using chart-releaser, hence bumping the chart version means a release gets shipped. I would like to also include other changes before, so reverting this.